apps/console/src/index.tsx

```diff
@@ -36,6 +37,32 @@ import Theme from "./theme";
 // Set the runtime config in the context.
 ContextUtils.setRuntimeConfig(Config.getDeploymentConfig());
 
+// Function to check the status of the Monaco CDN.
+// If the CDN is not available, the default CDN will be used.
+async function checkCDNStatus() {
```

Comment:
Lets use arrow functions and link a Git Issue with a `TODO`.
```suggestion
async function checkCDNStatus() {
```

Reply:
fixed with https://github.com/wso2/identity-apps/pull/6255/commits/c7d66702d38416b4e7c78e3a5ce8c98dc3c30d9a